lib/generators/suspenders/ci_generator.rb

```diff
@@ -0,0 +1,12 @@
+module Suspenders
+  module Generators
+    class CiGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/ci", __FILE__)
+
+      def cifiles
+        empty_directory ".github/workflows"
+        template "ci.yml", ".github/workflows/ci.yml"
```

Comment:
I wonder if we should add a [dependabot](https://github.com/rails/rails/pull/50508/files#diff-1182a749a11dc300cbc5af5e664598ab031c4b152cd2828ecae02d2d4f09168d) script too?